sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/board_detection.py

```diff
@@ -133,6 +133,8 @@ def get_tilt(self, model: Optional[CameraModel] = None) -> float:
         if model == self._cached_camera_model and self._cached_tilt is not None:
             return self._cached_tilt
 
+        # cSpell:ignore rvec, tvec
```

Comment:
Also acceptable in our local dictionary?

Reply:
Done!